python/lsst/ts/externalscripts/make_LOVE_stress_tests.py

```diff
@@ -0,0 +1,299 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["StressLOVE"]
+
+import asyncio
+import json
+
+import aiohttp
+import requests
+import yaml
+from astropy.time import Time
+from lsst.ts import salobj
+
+
+class ClientInstance:
```

Comment:
I suggest renaming this to avoid the word "Instance"; that's potentially very confusing to readers because when you instantiate a class the result is an instance (in this case an instance of ClientInstance). Perhaps LoveManagerClient?